lm_eval/jasquad/evaluate.py

```diff
@@ -0,0 +1,111 @@
+""" Official evaluation script for v1.1 of the SQuAD dataset. """
+
+import argparse
+import json
+import re
+import string
+import sys
+from collections import Counter
+
+
+try:
```

Comment:
I think this is not desirable in lm-eval-harness perspective. Because you would get the ImportError even if JSQuAD/JaQuAD are not used.